src/app/tile/panel/terra-tile-box-panel.component.ts

```diff
@@ -0,0 +1,147 @@
+import {
+    Component,
+    Input
+} from '@angular/core';
+import { TerraTileBoxInterface } from '../box/data/terra-tile-box.interface';
+/**
+ * @author mkunze
+ */
+@Component({
+               selector: 'terra-tile-box-panel',
+               styles:   [require('./terra-tile-box-panel.component.scss').toString()],
+               template: require('./terra-tile-box-panel.component.html')
+           })
+export class TerraTileBoxPanelComponent
+{
+    @Input() inputTileBoxList:Array<TerraTileBoxInterface>;
+    @Input() inputIsViewToggleable:boolean;
+    private _selectedTileBoxList:Array<TerraTileBoxInterface> = [];
+    
+    private draggedIndex:number;
+    private _viewStyle:boolean = false;
+    
+    constructor()
+    {
+        this.inputIsViewToggleable = false;
+    }
+    
+    private onBoxClick(ev, tile:TerraTileBoxInterface):void
+    {
```

Comment:
die variablen 'ev' in 'event' umbenennen. dann ist es deutlicher, wofür die da sind :)